### ISSUE TITLE
Variable type bool does not prompt for value

### ISSUE THREAD

**Issue by frank-hopkin-accrisoft** [gruntwork-io/boilerplate#201]
**Describe the bug**
In the readme, the [example](https://github.com/gruntwork-io/boilerplate?tab=readme-ov-file#example-creating-a-new-template) provides a bool type variable example and shows a prompt should come up for the end-user to provide the value. The prompt never comes up for any bool type variables.

**To Reproduce**
Use the [example from the README.md](https://github.com/gruntwork-io/boilerplate?tab=readme-ov-file#example-creating-a-new-template) . 

**Expected behavior**
The README.md provides expected behavior, where it shows when you run the boilerplate command, it prompts for the values for Title, WelcomeText, and ShowLogo.

```
boilerplate --template-url /home/ubuntu/website-boilerplate --output-folder /home/ubuntu/website-output

Title

  Enter a value [type: string]: Boilerplate Example

WelcomeText
  Enter the welcome text for the website

  Enter a value [type: string]: Welcome!

ShowLogo
  Should the website show the logo?

  Enter a [type: bool]: true

Generating /home/ubuntu/website-output/index.html
Copying /home/ubuntu/website-output/logo.png
```

**Actual behavior**
The boilerplate command only prompts for Title and WelcomeText.

**Additional context**
I am aware that using the --var argument achieves the desired functionality of providing a value, but the documentation shows the prompts should ask for the value. At the least,the documentation needs updated if this works as designed. As an end user, I would expect the prompt to ask for all variable values, not specific types.

**Comment by eric-Grunt** [comment]
Confirmed this bug with [boilerplate_linux_arm64](https://github.com/gruntwork-io/boilerplate/releases/download/v0.5.19/boilerplate_linux_arm64) and [boilerplate_windows_amd64.exe](https://github.com/gruntwork-io/boilerplate/releases/download/v0.5.19/boilerplate_windows_amd64.exe).

**Comment by frank-hopkin-accrisoft** [comment]
Forgot to provide environment, using [boilerplate_linux_amd64](https://github.com/gruntwork-io/boilerplate/releases/download/v0.5.19/boilerplate_linux_amd64)

**Comment by oredavids** [comment]
Related to #170

**Comment by Resonance1584** [comment]
Fixed in [v0.6.0](https://github.com/gruntwork-io/boilerplate/releases/tag/v0.6.0)

**Comment by frank-hopkin-accrisoft** [comment]
Confirmed fixed in v0.6.0
